conductr_cli/test/test_conduct_logs.py

```diff
@@ -58,9 +52,9 @@ def test_multiple_events(self):
 
         http_method.assert_called_with(self.default_url)
         self.assertEqual(
-            strip_margin("""|TIME            HOST        LOG
-                            |Today 12:54:36  10.0.1.232  [WARN] [04/21/2015 12:54:30.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-22] Association with remote system has failed.
-                            |Today 12:54:30  10.0.1.232  [WARN] [04/21/2015 12:54:36.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-26] Association with remote system has failed.
+            strip_margin("""|TIME                       HOST        LOG
+                            |2015-07-29T12:10:34+10:00  10.0.1.232  [WARN] [04/21/2015 12:54:30.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-22] Association with remote system has failed.
+                            |2015-07-29T12:10:39+10:00  10.0.1.232  [WARN] [04/21/2015 12:54:36.079] [doc-renderer-cluster-1-akka.remote.default-remote-dispatcher-26] Association with remote system has failed.
```

Comment:
Can I check if my understanding is correct please? So it should behave like:

Command | Renders
---------------|------------
`conduct events my-bundle` | Locale Time without Date
`conduct events my-bundle --date` | Locale Date Time
`conduct events my-bundle --utc` | UTC Time without Date
`conduct events my-bundle --date --utc` | UTC Date Time